### PR TITLE
Feat/ccn3 2334/send reminder email

### PR DESCRIPTION
**Pull request by jenniferbrown94**
adding health check reminder

### CODE REVIEW

File: application/health_check_reminders.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+        with utils.CronErrorContext():
+            log.info('Checking for applications that have been accepted x days ago')
+            send_second_reminder = generate_list_of_adults_not_completed_health_check(settings.SECOND_HEALTH_CHECK_REMINDER_THRESHOLD)
+            send_third_reminder = generate_list_of_adults_not_completed_health_check(settings.THIRD_HEALTH_CHECK_REMINDER_THRESHOLD)
+            log.info(send_second_reminder)
+            log.info(send_third_reminder)
+
+            # send second reminder email
+            for adult in send_second_reminder:
+                with utils.CronErrorContext():
```
Comment (LRydeard) [inline application/health_check_reminders.py]: Can you explain what this does?

File: application/health_check_reminders.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+                        applicant_first_name = record['first_name']
+                    else:
+                        applicant_name = ''
+                        applicant_first_name = ''
+
+                    adult_name = adult['first_name']
+                    adult_full_name = adult['first_name'] + " " + adult['last_name']
+
+                    log.info(str(datetime.now()) + ' - Sending second health check reminder: ' + str(adult_id))
+                    adult_template_id = 'a9cf45d9-ab54-4c49-afa1-59b7cebb334f'
```
Comment (LRydeard) [inline application/health_check_reminders.py]: Are these template_ids supposed to be the same across second and third reminder emails?

Comment (jenniferbrown94) [reply application/health_check_reminders.py]: currently the content for both emails is the same

File: application/health_check_reminders.py (hunk 1 above)
Comment (Thomas-95IS) [reply application/health_check_reminders.py]: It's a context manager for catching and logging errors in sub-sections, without terminating the entire program. See the base class for more info.

**Pull request merged**